packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -226,13 +226,23 @@ export class SummaryManager extends TypedEventEmitter<ISummaryManagerEvents> imp
                 async () => summarizer.run(clientId, this.summarizerOptions),
             );
         }).catch((error) => {
-            this.logger.sendErrorEvent({ eventName: "SummarizerException" }, error);
-            this.emit("summarizerWarning", error);
-
-            // Note that summarizer may keep going (like doing last summary).
-            // Ideally we await stopping process, but this code path is due to a bug
-            // that needs to be fixed either way.
-            this.stop("summarizerException");
+            // Most of exceptions happen due to container being closed while loading it, due to
+            // summarizer container loosing connection while load.
+            // Not worth reporting such errors as errors. That said, we might miss some real errors if
+            // we ignore blindly, so try to narrow signature we are looking for - skip logging
+            // error only if interactive container should no longer be summarizer (which in practice
```

Comment:
```suggestion
            // error only if interactive container should no longer be summarized (which in practice
```

Reply:
summairizer is actually the right term. Changing it slightly :)

Reply:
Ah... I get it 🤦‍♂️ It totally makes sense. Sorry for the back-and-forth